package.json

```diff
@@ -19,8 +19,8 @@
   "author": "Gregor Martynus (https://dev.to/gr2m)",
   "license": "MIT",
   "devDependencies": {
-    "@octokit/core": "^4.0.5",
-    "@pika/pack": "^0.5.0",
+    "@octokit/core": "^5.0.1",
+    "@pika/pack": "^0.3.7",
```

Comment:
why the downgrade of `@pika/pack`?

We replaced the no longer maintained pika packages with esbuild and tsc, see https://github.com/octokit/app.js/pull/420. It would be great if we could do the same here to keep the two aligned, to make it simpler for maintainers and contributors to cross between the two projects

Reply:
Hmm. I dont know why i downgraded. I like esbuild. Will rewrite it.

Reply:
Vielen Dank! 👍🏼